frontend/src/components/header/Header.vue

```diff
@@ -52,7 +53,7 @@ const props = defineProps({
                             v-if="authService.getters.isLoggedIn()"
                             @click="useProfileSidebarVisible.toggleVisible()"
                         >
-                            <Icon medium icon="AvatarIcon"></Icon>
+                            <UserAvatar />
```

Comment:
Same comment as previous,  maybe I am confused, is it better to use Icon component?